worker/src/RTC/Producer.cpp

```diff
@@ -1264,6 +1269,19 @@ namespace RTC
 				bufferPtr += extenLen;
 			}
 
+			// Proxy http://www.webrtc.org/experiments/rtp-hdrext/playout-delay
```

Comment:
Also move this block after video orientation extension to keep the order, please.

Reply:
Video orientation in this context is inside an `if (this->kind == RTC::Media::Kind::VIDEO)` block. Although Google's WebRTC only supports this extension for video, there is nothing in the standard that prevents this extension from being applied to audio packets.

Reply:
Well, but you are only adding support for it in video. See your changes in supportedCapabilities.

Reply:
So if we can support this extension for audio packets then we should add official support for it in audio as well (in Rust and Node).